public/doc/swagger-2-v0.0.1.yaml

```diff
@@ -781,6 +781,41 @@ paths:
           description: Source deleted
         404:
           description: Not found
+  "/tasks":
+    get:
+      summary: List Tasks
+      operationId: listTasks
+      description: Returns an array of Task objects
+      produces:
+      - application/json
+      responses:
+        200:
+          description: Tasks array
+          schema:
+            type: array
+            items:
+              "$ref": "#/definitions/Task"
+  "/tasks/{id}":
+    get:
+      summary: Show an existing Task
+      operationId: showTask
+      description: Returns a Task object
+      produces:
+      - application/json
+      parameters:
+      - name: id
+        in: path
+        description: ID of the resource to return
+        required: true
+        type: string
+        format: uuid
```

Comment:
id is a uuid?

Reply:
not actually

Reply:
Yeah I thought uuid was odd but every other `id` was `format: uuid` so I figured it was some swagger thing.  E.g.: https://github.com/ManageIQ/topological_inventory-api/blob/f29276eb7a40ed85642ef668cc71a17cc2236a4f/public/doc/swagger-2-v0.0.1.yaml#L38-L44